scipy/optimize/_differentialevolution.py

```diff
@@ -788,6 +797,7 @@ def __init__(self, func, bounds, args=(),
             self._updating = updating
 
         self.vectorized = vectorized
+        self.minimizer_kwargs = minimizer_kwargs or {}
```

Comment:
```suggestion
        self.minimizer_kwargs = minimizer_kwargs or {"method": "L-BFGS-B"}
```

Reply:
See my other comment below: this would duplicate the name of the default method.